lib/tasks/package_categories.rake

```diff
@@ -0,0 +1,52 @@
+namespace :goodcity do
+
+  # rake goodcity:add_package_categories
+  desc 'add_package_categories'
+  task add_package_categories: :environment do
+
+    categories = {
+      "Electrical" => ["Computer", "Household", "Kitchen", "Other electrical",
+        "Audio visual", "Office appliances"],
+
+      "Furniture" => ["Baby / child", "Beds", "Desk", "Seating", "Shelves",
+        "Storage", "Table", "Other furniture", "Office furniture", "Sets"],
+
+      "Household" => ["Appliances", "Baby", "Textiles / clothes",
+        "Kitchen, bedroom, bathroom, dining",  "Other household"],
+
+      "Small goods & bulk items" => ["Books", "Food", "Glasses", "Stationery",
+        "Tools", "Toys", "Medical", "Clothing & accessories",
+        "Mixed / miscellaneous", "Household textiles"],
+
+      "Recreation" => ["Music", "Other", "Playground", "Sport", "Bicycles"]
+    }
```

Comment:
@swatijadhav This would be great in a yaml file so it's easier to use in ```factory_girl``` tests. See the other taxonomies for the same approach. e.g. https://github.com/crossroads/api.goodcity/blob/master/spec/factories/sequences.rb#L21